drivers/soundwire/stream.c

```diff
@@ -1270,6 +1259,17 @@ static int sdw_slave_port_alloc(struct sdw_slave *slave,
 	return 0;
 }
 
+static int sdw_slave_port_is_valid_range(struct device *dev, int num)
+{
+	if (!SDW_VALID_PORT_RANGE(num)) {
+		dev_err(dev,
+			"SoundWire: Invalid port number :%d\n", num);
```

Comment:
in one line ? dev_err(dev, "SoundWire: Invalid port number :%d\n", num);

Reply:
fixed